packages/eslint-plugin-typescript/src/rules/package/package-force-private.rule.ts

```diff
@@ -0,0 +1,10 @@
+import { Rule } from 'eslint';
+import { jsonRule } from '../json-util';
+
+export const PackagForcePrivateRule: Rule.RuleModule = {
+    create: (context) => jsonRule(context, { key: 'private', expectedValue: true }, 'package.json'),
```

Comment:
I think it would be valuable to have a configuration named `ignorePublished` which by default is true.

This then would ignore all packages that do contain a [`publishConfig`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#publishconfig), because then it can be assumed that these packages should be published.

Reply:
Addded the config and another test case for the behavior